app/helpers/application_helper.rb

```diff
@@ -0,0 +1,10 @@
+module ApplicationHelper
+  # Returns the full title on a per-page basis.
+def full_title(page_title = '')
+  base_title = "Bamgram" 
+  if page_title.empty?
+    base_title
+  else
+    page_title + " | " + base_title end
```

Comment:
tiny detail - playing rubocop here but do you want to move the 'end' on line 8 to the line below?